src/main/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZRestCommunicator.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) 2021 AVI-SPL Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.communicator.aver.ptz;
+
+import com.avispl.symphony.dal.communicator.RestCommunicator;
+
+/**
+ * This class is used to getting monitoring properties:
+ *
+ * - Device Mfg
+ * - Device Model
+ * - Serial Number
+ * - Firmware Version
+ *
+ * @author Harry
+ * @version 1.0
+ * @since 1.0
+ */
+public class AverPTZRestCommunicator extends RestCommunicator {
+	private String deviceMfg;
+	private String deviceModel;
+	private String deviceSerialNumber;
+	private String deviceFirmwareVersion;
+
+	/**
+	 * Empty constructor
+	 */
+	public AverPTZRestCommunicator() {
```

Comment:
If you don't have the plan to use it in the future, please remove it

Reply:
ok, I will remove it